ui/public/style.css

```diff
@@ -33,6 +33,15 @@ body {
     100% { transform: rotate(360deg); }
 }
 
+/* Records list table */
+
+td {
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  max-width: 65px;
+}
+
```

Comment:
Do you also want to add a max-width to the table itself? It seems from your screenshot that it's displaying below the filter selection?